api/client/webclient/webclient.go

```diff
@@ -68,6 +68,9 @@ type Config struct {
 	Timeout time.Duration
 	// TraceProvider is used to retrieve a Tracer for creating spans
 	TraceProvider oteltrace.TracerProvider
+	// UpdateGroup is used to modulate the webapi response based on the
```

Comment:
Nit: "modulate" is a bit of an odd word. Perhaps clearer to say "vary" or "change".